fuel-vm/src/interpreter.rs

```diff
@@ -152,6 +145,14 @@ impl<S, Tx> Interpreter<S, Tx> {
     }
 }
 
+pub(crate) fn is_wrapping(flag: Reg<FLAG>) -> bool {
+    *flag & 0x02 == 0x02
```

Comment:
maybe we should use constants for these flag values? Or bring back the bitfield library?

Reply:
I guess https://github.com/FuelLabs/fuel-vm/pull/363 will cause a merge conflict that should be resolved in favor of @Dentosal change=)